go.sum

```diff
@@ -178,6 +178,7 @@ github.com/opsgenie/opsgenie-go-sdk-v2 v1.0.6 h1:jbPcx0GlwyDkFqFPVQQ5SXCHMVmftLo
 github.com/opsgenie/opsgenie-go-sdk-v2 v1.0.6/go.mod h1:f0ezb0R/mrB9Hpm5RrIS6EX3ydjsR2nAB88nYYXZcNY=
 github.com/opsgenie/opsgenie-go-sdk-v2 v1.0.7-0.20200406122459-ce3f70f22ab7 h1:+cTGQp1QRUWH+YGgh2b4sO/Y5g86PTpVEIpVFnEor6E=
 github.com/opsgenie/opsgenie-go-sdk-v2 v1.0.7-0.20200406122459-ce3f70f22ab7/go.mod h1:VOkJ7STzYj+nXRhMcBTcmt8uZZ17KZKJdZtJpgHLbT8=
+github.com/opsgenie/opsgenie-go-sdk-v2 v1.1.1 h1:0y6RTB5SNcf0lhdVyJFPU/bTagUlToKpW4AGWMB7ZBw=
```

Comment:
could you change this to master 
my pr has been merged into go-sdk-v2 and get method will work on the master branch

Reply:
wrong file my bad, in the go.mod file*

Reply:
@ffahri Just did but I'm getting some issues with `notification` resource:

![image](https://user-images.githubusercontent.com/8523281/86031312-87336680-b9ea-11ea-9b59-a3262d250f15.png)

Reply:
```bash
❯ make test
==> Checking that code complies with gofmt requirements...
go test -i $(go list ./... |grep -v 'vendor') || exit 1
# github.com/terraform-providers/terraform-provider-opsgenie/opsgenie
opsgenie/resource_opsgenie_notification_policy.go:520:22: cannot use config["until_minute"].(int) (type int) as type *int in assignment
opsgenie/resource_opsgenie_notification_policy.go:521:20: cannot use config["until_hour"].(int) (type int) as type *int in assignment
make: *** [GNUmakefile:12: test] Error 1

```

Reply:
```
		action.UntilMinute = config["until_minute"].(*int)
		action.UntilHour = config["until_hour"].(*int)
```
should do the trick

Reply:
@ffahri Cool, dep updated and pushed to the PR. As for the `Service API` fields I noticed Sat that the `sdk` didn't support the `tags`  field that's why I left it out.

Reply:
last one 😄 
```
cannot use config["until_minute"].(int) (type int) as type *int in assignment
```

you forgot to push policy.go on the vendor

Reply:
@ffahri Is this what you mean https://github.com/terraform-providers/terraform-provider-opsgenie/pull/115/commits/b7cd2829f87a8f9bf79cd0af22bcbb70d8a62ca6

Reply:
yes, but you can remove it since its nothing related to this pr, we can add this after this pr

Reply:
completeTest contains 
```
	visibility = "%s"

```
after we removed that e2e will pass and we can merge

Reply:
@ffahri Cool. As you suggested, removed the `b7cd282` commit regarding vendoring. 

Thanks for catching the `visibility` field on the e2e tests as well